PHASE_1/API_SourceCode/src/interfaces/IApiResponses.ts

```diff
@@ -24,9 +25,26 @@ export interface IArticlesDumpSuccessResponse {
   articles: Article[];
 }
 
+export interface IArticleSpecificSuccessResponse {
+  article: Article;
+}
+
+export interface IArticleRequestHeaders {
```

Comment:
for specific report and article requests, we don't need interfaces since they don't have any request body, and especially here since we don't have any specific headers either?